drivers/gles3/storage/texture_storage.cpp

```diff
@@ -1566,6 +1566,36 @@ void TextureStorage::_update_render_target(RenderTarget *rt) {
 		bool use_multiview = rt->view_count > 1 && config->multiview_supported;
 		GLenum texture_target = use_multiview ? GL_TEXTURE_2D_ARRAY : GL_TEXTURE_2D;
 
+		/* Back FBO */
+		glGenFramebuffers(1, &rt->copy_fbo);
+		glBindFramebuffer(GL_FRAMEBUFFER, rt->copy_fbo);
+
+		if (rt->copy_color == 0) {
+			glGenTextures(1, &rt->copy_color);
+			glBindTexture(texture_target, rt->copy_color);
+
+			glTexImage2D(texture_target, 0, rt->color_internal_format, rt->size.x, rt->size.y, 0, rt->color_format, rt->color_type, nullptr);
```

Comment:
This needs multiview support, but we can add that in later. In theory it should be enough to switch between 2D and 3D textures if `use_multiview` is true and use `rt->viewcount` as the z component.